cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -975,19 +977,55 @@ def _validate(self, instance_type, dcv_enabled, allowed_ips, port, os, architect
                 )
 
 
-class IntelHpcOsValidator(Validator):
+class IntelHpc2018OsValidator(Validator):
     """Intel HPC OS validator."""
 
     def _validate(self, os: str):
-        allowed_oses = ["centos7"]
+        allowed_oses = SUPPORTED_OSES_FOR_FEATURE[Feature.INTEL_HPC_SPECIFICATION_2018]
         if os not in allowed_oses:
             self._add_failure(
-                "When enabling intel software, the operating system is required to be set "
+                "When enabling intel HPC platform 2018, the operating system is required to be set "
                 f"to one of the following values : {allowed_oses}.",
                 FailureLevel.ERROR,
             )
 
 
+class IntelOneApiToolkitsBootstrapTimeValidator(Validator):
+    """Intel OneApi Toolkits Bootstrap time Validator."""
+
+    def _validate(self):
+        self._add_failure(
+            "Cluster creation will take ~30 minutes, because installing Intel OneAPI Base Toolkit takes ~10 minutes.",
+            FailureLevel.INFO,
+        )
+
+
+class IntelOneApiToolkitsOsValidator(Validator):
+    """Intel OneApi Toolkits Os Validator."""
+
+    def _validate(self, os: str):
+        supported_oses = SUPPORTED_OSES_FOR_FEATURE[Feature.INTEL_ONE_API_BASE_TOOLKIT]
```

Comment:
Why are we checking only base toolkit and not hpc toolkit?

Reply:
Intel HPC toolkit requires Intel Base toolkit. So this check covers all